spec/requests/api/v1/relationships/invoice_items_relationships_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe 'Invoice Items Relationship Endpoints' do
+  before(:each) do
+    invoice_items = create_list(:invoice_item, 20)
+    @invoice_item = invoice_items.sample
+  end
+  it 'returns the associated invoice' do
+
+    get "/api/v1/invoice_items/#{@invoice_item.id}/invoice"
+
+    found = JSON.parse(response.body)
+
+    expect(found).to be_a(Hash)
+    expect(found["id"]).to eq(@invoice_item.invoice_id)
+    expect(found).to have_key("merchant_id")
+    expect(found).to have_key("customer_id")
+    expect(found).to have_key("status")
+  end
```

Comment:
I've been wondering if maybe we could clean up these tests by testing;
expect(found.keys).to eq(["id","merchant_id","customer_id","status"])

And test all of them in one line